tgui/packages/tgui/interfaces/PaiInterface.tsx

```diff
@@ -395,9 +409,10 @@ const RecordsDisplay = (props, context) => {
       fill
       scrollable>
       <Table>
-        {convertedRecords && convertedRecords.map((record) => {
-          return <RecordRow key={record} record={record} />;
-        })}
+        {convertedRecords &&
+          convertedRecords.map((record) => {
```

Comment:
you can replace this with `convertedRecords?.map` (but you must be sure convertedRecords is NULL and not FALSE, you should make sure of it in ui_data)